client/src/components/ChefSearch.js

```diff
@@ -0,0 +1,194 @@
+import React, { useState, useEffect } from "react";
+import API from "api/index";
+import {
+  Grid,
+  Paper,
+  Button,
+  Box,
+  Input,
+  Typography,
+  Chip,
+  Card,
+  CardActionArea,
+  CardActions,
+  CardMedia,
+  CardContent,
+} from "@material-ui/core";
+import { LocationOn, Clear } from "@material-ui/icons";
+import ChefFilters from "components/ChefFilters";
+import { makeStyles } from "@material-ui/core/styles";
+import { addressToCoords, coordsToAddress } from "api/googleMaps";
+import chefImage from "images/chef.png";
+import chefPlaceholder from "images/chefPlaceholder.jpeg";
+
+const ChefSearch = ({ coords }) => {
+  const [chefs, setChefs] = useState([
+    {
+      name: "Sushi Chef",
+      cuisine: ["Japanese"],
+      location: "Toronto, Canada",
+      description:
+        "Sushi Master. 20 Years of experience working under Sushi Masters in Japan.",
+      img: chefImage,
+    },
+    {
+      name: "Pasta Chef",
+      cuisine: ["Italian", "American"],
+      location: "Toronto, Canada",
+      description: "Pasta Master",
+      img: chefImage,
+    },
+  ]);
+  const [cuisineTypes, setCuisineTypes] = useState([
+    "Japanese",
+    "Chinese",
+    "American",
+    "French",
+    "Mexican",
+    "Italian",
+  ]);
+  const [userAddress, setUserAddress] = useState("");
+  const [userCoordinates, setUserCoordinates] = useState(null);
+  const [selectedCuisines, setSelectedCuisines] = useState([]);
+  const [distanceFilter, setDistanceFilter] = useState(null);
+
+  const classes = useStyles();
+  useEffect(() => {
+    // do api call
+    searchChefs();
+    // to change later
+    setDummyData();
```

Comment:
Are you planning on removing this dummy data in a future PR?

Reply:
Yup, just had to put some stub data for cuisines